core/server/data/migrations/versions/4.33/02-add-default-free-product.js

```diff
@@ -0,0 +1,37 @@
+const {createTransactionalMigration} = require('../../utils');
+const ObjectID = require('bson-objectid');
+const {slugify} = require('@tryghost/string');
+const logging = require('@tryghost/logging');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const [result] = await knex
+            .count('id', {as: 'total'})
+            .from('products')
+            .where('type', 'free');
+
+        if (result.total !== 0) {
+            logging.warn(`Not adding default free tier, a free tier already exists`);
+            return;
+        }
+
+        const name = 'Free';
+        const id = ObjectID().toHexString();
+
+        logging.info(`Adding tier "${name}"`);
+        await knex('products')
+            .insert({
+                id: id,
+                name: name,
+                type: 'free',
+                slug: slugify(id),
```

Comment:
Do we want to be `slugify`ing the `id`? Would the `name` not make more sense? 🤔

Reply:
Yeah that's what we originally did, but we realized that might actually have conflicts for anyone that might have already created a paid tier with name `Free`, which is unlikely but possible 😬  We don't use slugs anywhere right now for a product, so to be on a safe side with the migration running across all sites that might have some used `slug`, we went with using the `id` as the slug to keep it unique

Reply:
Ah!